class.jetpack.php

```diff
@@ -533,6 +534,9 @@ private function __construct() {
 			add_action( 'init', array( 'Jetpack_Iframe_Embed', 'init' ), 9, 0 );
 			require_once JETPACK__PLUGIN_DIR . '_inc/lib/class.jetpack-keyring-service-helper.php';
 			add_action( 'init', array( 'Jetpack_Keyring_Service_Helper', 'init' ), 9, 0 );
+
+			$this->tracking = new Tracking( 'jetpack' );
+			add_action( 'init', array( $this, 'track_jetpack_usage' ) );
```

Comment:
Currently, `Jetpack::init` is hooked on `init` itself. So I'd expect it is too late to hook this on the same hook?

Reply:
seem like we do it on `init` and before `plugins_loaded` 😱